correios/models/address.py

```diff
@@ -280,7 +281,15 @@ def zip_code_display(self) -> str:
 
     @property
     def basic_address(self):
-        return "{}, {}".format(self.street, self.number)
+        number = self.number
+        if self.complement:
+            number = "{} {}".format(number, self.complement)
+
+        address = "{}, {}, {}".format(self.street, number, self.neighborhood)
+        if len(address) <= 60:
```

Comment:
this check is necessary to avoid a blank line on short addresses, e.g.:

without the check:
![sem_len](https://cloud.githubusercontent.com/assets/3208493/22076667/31ef904a-dd98-11e6-82a3-70ab0944e164.png)

with it:
![com_len](https://cloud.githubusercontent.com/assets/3208493/22076673/384c4848-dd98-11e6-9d28-fe79e86edd90.png)

Perhaps there is a better way to this. But I can't think of anything else.